### PR TITLE
DOC: update tutorial to use asyncio instead of asyncore

### LINKED ISSUE #85: How to read events using asyncio
in the documentation, there is an example to read events using asyncore with a note along the example suggesting to use asyncio with no explanation, and am also need explanation what is the need of reading events using asyncore.

### PR DESCRIPTION
**Pull request by tacaswell**
Closes #85

### CODE REVIEW

**Comment by tacaswell** [comment]
I can split this up into several PRs if you prefer.

File: evdev/device.py [hunk 1]
```diff
@@ -12,6 +12,9 @@
 except ImportError:
     from evdev.eventio import EventIO, EvdevError
 
+import sys
+if sys.version_info > (3,):
+    buffer = memoryview
```
Comment (gvalkov) [inline evdev/device.py]: I found out that `memoryview(...).tobytes()` works for both Py2 and Py3.

**Comment by gvalkov** [comment]
Thanks @tacaswell ! Your contribution is appreciated. I just need to sit down and see for myself how the asyncio API will be changing before rolling out a release with your fixes.

**Comment by gvalkov** [comment]
Released as part of evdev 1.0.0.

**Comment by tacaswell** [comment]
Your welcome!

**Pull request merged**